Gemfile.development_dependencies

```diff
@@ -1,16 +1,15 @@
 # frozen_string_literal: true
 
-gem "shakapacker", "7.0.1"
+gem "shakapacker", "7.2.1"
 gem "bootsnap", require: false
-gem "rails", "~> 7.0", ">= 7.0.1"
+gem "rails", "~> 7.1"
```

Comment:
any updates needed for the Rails upgrade?

Reply:
I didn't notice anything to apply to this project.